packages/stylelint-plugin/src/rules/no-invalid-equations.ts

```diff
@@ -0,0 +1,165 @@
+import { Declaration, Root } from "postcss"
+import postcssValueParser, { WordNode } from "postcss-value-parser"
+import {
+  kaizenVariableUsedNextToOperatorMessage,
+  negatedKaizenVariableMessage,
+  tokenNotInterpolatedInCalcMessage,
+} from "../messages"
+import { isOperator } from "../patterns"
+import { Options } from "../types"
+import { isVariable, parseVariable, stringifyVariable } from "../variableUtils"
+import { walkDeclsWithKaizenTokens, walkWithParent } from "../walkers"
+
+export const noInvalidEquationsRuleName = "no-invalid-equations"
+export const noInvalidEquationsOnDeclaraion = (
```

Comment:
This doesn't appear to be used in other modules, so we probably shouldn't export it